README.md

```diff
@@ -30,8 +30,8 @@ repositories {
 
 ```gradle
 dependencies {
-  debugImplementation 'fr.o80.chucker:library:2.0.1'
-  releaseImplementation 'fr.o80.chucker:library-no-op:2.0.1'
+  debugImplementation 'fr.o80.chucker:library:2.1.0'
```

Comment:
Are you sure we want to update this as well?
If we're publishing `-SNAPSHOT` versions on maven, the stable version (`2.1.0`) will not be yet available right?

I'd rather keep `-SNAPSHOT` only on `gradle.properties` and we update the README as soon as we release a new version

Reply:
I've merged your work about Jitpack+SNAPSHOT, what do I have to do with this change?

Reply:
So the idea is to merge this, as soon as we're ready with `2.1.0` right?
If so, just remove the `-SNAPSHOT` from the `VERSION_NAME` field and everything will be fine.

Reply:
Ok, I've removed the "-SNAPSHOT" part of the version in `gradle.properties`.